recipe/meta.yaml

```diff
@@ -10,8 +10,10 @@ source:
   sha256: 8781ab15ce13fcddcb7df8fd23755a5cb6e4ab7d1a9bdbd741128b990b425496
 
 build:
+  run_exports:
+    - {{ pin_subpackage('libestools', min_pin='x.x', max_pin='x.x') }}
```

Comment:
You need to copy this build/run_export section to every output that could be used as a host dependency downstream.

This is also an unusual pinning. Probably, you want to use the default min_pin, and only specify the max_pin.

Reply:
Not sure if I did it correctly. Can you please verify?